### ISSUE TITLE
trouble when using docker to train lora

### ISSUE THREAD

**Issue by YachiZhang** [bmaltais/kohya_ss#2349]
I use build the docker to train lora. but different from the windows, it doesn't show the training log .like the following:
![screenshot-20240419-214227](https://github.com/bmaltais/kohya_ss/assets/24383604/f0b1e03d-d3bd-42d3-863a-837e62e8106e)
Another problem, when I start the training already, if I close the page and reopen, I can not stop the training through the ui.

**Comment by bmaltais** [comment]
> Another problem, when I start the training already, if I close the page and reopen, I can not stop the training through the ui.

Yeah... This will happen... not sure how to best fix that... maybe by always desplaying both buttons in headless mode?

**Comment by YachiZhang** [comment]
> > Another problem, when I start the training already, if I close the page and reopen, I can not stop the training through the ui.
> 
> Yeah... This will happen... not sure how to best fix that... maybe by always desplaying both buttons in headless mode?

Thanks~, and because it does not have the terminal like windows , if I want stop
the training, I should "kill $pid"?

**Comment by bmaltais** [comment]
At the moment yes.

**Comment by bmaltais** [comment]
I have implemented a fix where it will show both buttons in headless mode.

**Comment by YachiZhang** [comment]
> I have implemented a fix where it will show both buttons in headless mode.

Thank you very much~
